openroad.bzl

```diff
@@ -666,15 +667,15 @@ def build_openroad(
 
     outs = init_output_dict(all_stages, platform, out_dir, variant, name)
 
-    x = map(lambda ext: map2(lambda m: "//:results/" + platform + "/%s/%s/%s.%s" % (m, macro_variants.get(m, macro_variant), m, ext), macros), ["lef", "lib"])
+    x = map(lambda ext: map2(lambda m: "//" + Label(m).package + ":results/" + platform + "/%s/%s/%s.%s" % (Label(m).name, macro_variants.get(Label(m).name, macro_variant), Label(m).name, ext), macros), ["lef", "lib"])
     macro_lef_targets, macro_lib_targets = x
     # macro_gds_targets = map(lambda m: "//:results/" + platform + "/%s/%s/6_final.gds" % (m, macro_variants.get(m, macro_variant)), macros)
 
     # Get only the first source from constraints
     io_constraints_args = ["IO_CONSTRAINTS=$$(echo '$(locations " + io_constraints + ")' | cut -d' ' -f 1)"] if io_constraints != None else []
 
-    ADDITIONAL_LEFS = " ".join(map(lambda m: "$(RULEDIR)/results/" + platform + "/%s/%s/%s.lef" % (m, macro_variants.get(m, macro_variant), m), macros))
-    ADDITIONAL_LIBS = " ".join(map(lambda m: "$(RULEDIR)/results/" + platform + "/%s/%s/%s.lib" % (m, macro_variants.get(m, macro_variant), m), macros))
+    ADDITIONAL_LEFS = " ".join(map(resolve_path, macro_lef_targets))
+    ADDITIONAL_LIBS = " ".join(map(resolve_path, macro_lib_targets))
```

Comment:
`ADDITIONAL_LIBS` and `ADDITIONAL_LEFS` has to be a list of paths, not Bazel targets. They are passed straight to the ORFS, so Bazel target are not resolved